### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by StephenOTT**

### CODE REVIEW

**Review by stefanobaghino-da — APPROVED** [review]

**Comment by stefanobaghino-da** [comment]
@StephenOTT Thanks for your contribution. Your PR is missing change log entry (which can be empty since the PR is trivial). I recommend you read the contribution guidelines before submitting a PR for review, in particular with regards to the [PR checklist](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md#pull-request-checklist) which also appears as a comment in the template for the PR you opened.

tl;dr please run the following commands on your machine
```
git commit --amend --no-edit -m 'fix typo

changelog_begin
changelog_end'
git push --force
```

We use this system to collect changes with a visible external impact that needs to be reflected in our release notes without having a single file that needs merging for every contribution and made this mandatory even with an empty message as a reminder to all contributors.

**Comment by garyverhaegen-da** [comment]
Docs have moved to the digital-asset/docs.daml.com repo and it looks like this has been fixed over there. Apologies @StephenOTT for the time this took.
